Resources/doc/features.md

```diff
@@ -0,0 +1,12 @@
+Features
+===========
+
+Supports multiple entity managers 
+-------------
+
+Configure multiple entity managers according to http://symfony.com/doc/current/cookbook/doctrine/multiple_entity_managers.html
```

Comment:
This isn't the best example of how an entity manager would be automatically determined from an entity. The prefix and dir config options below are very important in determining the EM associated with the entity.


```
entity_managers:
    default:
        connection: default
        mappings:
            BookSellingBundle:
                type: annotation
                prefix: BookSellingBundle\Entity\Library
                dir:  Entity/Library
            EcentriaRestBundle: # we want to use this EM for transactions
                type: annotation
                dir:  Entity/

    other:
        connection: other
        mappings:
            BookSellingBundle:
                type: annotation
                prefix: BookSellingBundle\Entity\Customer
                dir:  Entity/Customer
```